android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/AccountOutParser.java

```diff
@@ -42,26 +42,30 @@ public static AccountOut parse(String rpcResult) {
                 int xmlHeaderEnd = rpcResult.indexOf("?>");
                 outResult = rpcResult.substring(0, xmlHeaderStart);
                 outResult += rpcResult.substring(xmlHeaderEnd + 2);
-            } else
+            }
+            else {
                 outResult = rpcResult;
+            }
 
             AccountOutParser parser = new AccountOutParser();
             Xml.parse(outResult, parser);
             return parser.getAccountOut();
-        } catch (SAXException e) {
+        }
+        catch (SAXException e) {
             return null;
         }
     }
 
     @Override
     public void startElement(String uri, String localName, String qName, Attributes attributes) throws SAXException {
         super.startElement(uri, localName, qName, attributes);
-        if (localName.equalsIgnoreCase("error_num") ||
-                localName.equalsIgnoreCase("error_msg") ||
-                localName.equalsIgnoreCase("authenticator")) {
-            if (mAccountOut == null)
+        if (localName.equalsIgnoreCase("error_num") || localName.equalsIgnoreCase("error_msg") ||
```

Comment:
This is a type of change that makes things worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.